controller.sh

```diff
@@ -39,13 +41,13 @@ case "$1" in
     UNIFI_API="https://localhost:${DOCKER_HTTPS_PORT}/" \
     UNIFI_ACC_WLAN_CONCURRENCY="4" \
     UNIFI_INSECURE="true" \
-    go test -v -cover -count 1 ./internal/provider
+    go test -v -cover -count 1 -run 'WLAN|AP' ./internal/provider
```

Comment:
What does this do?

Reply:
Oops accidentally left that in, I was just contraining the tests to WLAN/AP tests, need to rerun this with all tests.